From 3da40368ad87d36ca2434b7086684778b1e5a8ab Mon Sep 17 00:00:00 2001
From: Ben Weinstein-Raun <root@benwr.net>
Date: Sat, 20 Jan 2024 21:00:54 -0500
Subject: [PATCH] don't use command -p

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 2d006da..000d34e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -285,7 +285,7 @@ lib_libtarsnap_a_CPPFLAGS=				\
 LIBTARSNAP_A=	lib/libtarsnap.a
 
 cpusupport-config.h:
-	( export CC="${CC}"; export CFLAGS="${CFLAGS}"; command -p sh $(srcdir)/libcperciva/cpusupport/Build/cpusupport.sh "$$PATH" ) > cpusupport-config.h.tmp && command -p mv cpusupport-config.h.tmp cpusupport-config.h
+	( export CC="${CC}"; export CFLAGS="${CFLAGS}"; sh $(srcdir)/libcperciva/cpusupport/Build/cpusupport.sh "$$PATH" ) > cpusupport-config.h.tmp && mv cpusupport-config.h.tmp cpusupport-config.h
 
 # crypto_aesctr_shared.c is in this list because it can't be included in the
 # _SOURCES because it should only be included as part of another translation
-- 
2.43.0

